controllers/overrides_test.go

```diff
@@ -0,0 +1,87 @@
+package controllers
+
+import (
+	"context"
+	"testing"
+
+	"github.com/go-logr/logr"
+	ranv1alpha1 "github.com/openshift-kni/cluster-group-upgrades-operator/api/v1alpha1"
+	utils "github.com/openshift-kni/cluster-group-upgrades-operator/controllers/utils"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestOverrides_getOverrides(t *testing.T) {
+	testcases := []struct {
+		name     string
+		objectNs string
+		readNs   string
+		wrData   map[string]string
+		rdData   map[string]string
+	}{
+		{
+			name:     "Overrides exist",
+			objectNs: "test",
+			readNs:   "test",
+			wrData: map[string]string{
+				"precache.image":                "quay.io/test_images/pre-cache:latest",
```

Comment:
Will this be updated to quay.io/openshift-kni/cluster-group-upgrades-operator-precache once the upstream repo and mirroring is setup, or is this test_image special? If so, maybe add a TODO comment here

Reply:
Since this is the overrides test, I just changed all the image names to be very generic. I think it will be less confusing if the names would not resemble the real repositories and images